examples/hello_sessions_c.c

```diff
@@ -0,0 +1,24 @@
+#include "mpi.h"
+#include <stdio.h>
+
+/*
+ * Simple test to demonstrate several aspects of MPI 4 Sessions and related
+ * functionality.  See sections 11.3 and 11.4 of the MPI 4 standard for more
+ * details.
+ */
+
+int main(int argc, char** argv) {
+  MPI_Info info;
+  MPI_Session s1, s2;
+
+#if 0
+/* need PR https://github.com/open-mpi/ompi/pull/12868 to be merged in
+ * before this can be uncommented.
+ */
+  MPI_Info_create(&info);
+#endif
+  MPI_Session_init(MPI_INFO_NULL, MPI_ERRORS_RETURN, &s1);
+  MPI_Session_finalize(&s1);
+  MPI_Session_init(MPI_INFO_NULL, MPI_ERRORS_RETURN, &s2);
+  MPI_Session_finalize(&s2);
+}
```

Comment:
Should put a `return 0` here.

Reply:
The cherry-pick of https://github.com/open-mpi/ompi/pull/12888 back to v5.0.x will address that first comment.
I'll add the return 0 in a PR to main first.  Maybe I'll cherry-pick the commit from #12888 on to this PR.

Reply:
return 0 added